ci/jobs/test_unit.sh

```diff
@@ -0,0 +1,27 @@
+#! /bin/bash
+
+OWNER_BUILD_DIR=`dirname ${GITHUB_WORKSPACE}`
+echo OWNER_BUILD_DIR is $OWNER_BUILD_DIR
+
+source ${GITHUB_WORKSPACE}/internal_tests/pytests/minimum_pytest.docker.sh
+#export TRAVIS_INPUT_BASE=${METPLUS_TEST_INPUT_BASE}
+#export TRAVIS_OUTPUT_BASE=${METPLUS_TEST_OUTPUT_BASE}
+export TRAVIS_INPUT_BASE=${METPLUS_TEST_INPUT_BASE/$DOCKER_DATA_DIR/$OWNER_BUILD_DIR}
```

Comment:
Is this syntax really correct?
${METPLUS_TEST_INPUT_BASE/$DOCKER_DATA_DIR/$OWNER_BUILD_DIR}

Reply:
That command takes METPLUS_TEST_INPUT_BASE and replaces the text in $DOCKER_DATA_DIR with the text in $OWNER_BUILD_DIR. This was done since the minimum_pytest.docker.sh file lists locations on docker and we needed to find the Travis equivalent. It may not be necessary because the paths on each environment could be the same potentially. There are a few places where I started trying to understand and improve what was set up but decided to focus on recreating what we currently have running. Yes, many references to TRAVIS should be changed. I figured it would make more sense to get it working and revisit/refactor these scripts at a later date so I am able to continue the development on the tasks I was working on before this came up.